src/views/Home/AlertsView/AlertsAside/CountryDetail/index.tsx

```diff
@@ -116,19 +116,19 @@ function CountryDetail(props: Props) {
                         variant="tertiary"
                     >
                         <TabList className={styles.tabList}>
-                            <Tab name="alerts">
+                            <Tab name={strings.countryDetailsAlertName}>
                                 {strings.alertsAsideAlert}
                             </Tab>
-                            <Tab name="admin1">
+                            <Tab name={strings.countryDetailsAdmin1Name}>
                                 {strings.alertsAsideAdmin}
                             </Tab>
                         </TabList>
-                        <TabPanel name="alerts">
+                        <TabPanel name={strings.countryDetailsAlertName}>
                             <CountryAlerts
                                 countryId={countryId}
                             />
                         </TabPanel>
-                        <TabPanel name="admin1">
+                        <TabPanel name={strings.countryDetailsAdmin1Name}>
```

Comment:
we shouldn't use translated strings for variables. We should only translate the strings. These strings are visible ones. This will break the application when the language is changed.